### ISSUE TITLE
Screen is not rotated when the indicator menu is open in landscape

### ISSUE THREAD

**Issue by lightonflux** [ubports/ubuntu-touch#1251]
- Device: MX4
- Channel: Stable?
- Build: OTA-10

### Steps to reproduce

1. Unlock the phone
1. Disable the rotation lock if enabled
1. Rotate the phone into landscape oritation
1. Pull down the indicator menu
1. Rotate the phone 180°
1. Notice that the UI is not rotated
1. Close the indicator menu
1. Observe the rotation of the UI

### Expected behavior
The UI should be rotated, because the screen rotation lock is disabled.

### Actual behavior
UI is not rotated when the indicator menu is open. Does not matter which indicator is currently being viewed.

**Comment by lightonflux** [comment]
This also happens in portrait orientation.

**Comment by UniversalSuperBox** [comment]
Huh, I thought there was another report for this already.

It appears this is not by design, but is a side effect of some other part of unity8 implementation: https://blueprints.launchpad.net/ubuntu/+spec/appdev-1303-device-rotation

> The issues with the current rotation support:
> - the indicator panel does not rotate and provides an awkward experience

There may be more recent documentation stating that this was accepted into design, and if so I'd like to see it.

**Comment by jug007** [comment]
If this was by design: There is a good reason for the screen to rotate, with the indicator menu being open. If you turn off the rotation lock, you will most likely want to immediately rotate the screen. The indicator menu preventing you from doing that, would be annoying in that situation.

**Comment by mateosalta** [comment]
You can disable, but there are other graphical bugs, like the indicator pannel not populating, and changing pages if you have your finger down, ect. I just found the line that disables. https://github.com/ubports/unity8/blob/xenial/qml/Shell.qml#L90

**Comment by UniversalSuperBox** [comment]
Yes, it appears rotation is disabled because the Indicators don't re-flow unless they are closed and reopened.

**Comment by UniversalSuperBox** [comment]
@mateosalta, no. Ayatana and Ubuntu Indicators are nearly direct replacements of each other, but they are only the *models* of what is displayed by Lomiri. Lomiri is the thing that creates the presentation and decides how to reflow things. The fix is in Lomiri, not in the indicators.

**Comment by mateosalta** [comment]
> @mateosalta, no. Ayatana and Ubuntu Indicators are nearly direct replacements of each other, but they are only the _models_ of what is displayed by Lomiri. Lomiri is the thing that creates the presentation and decides how to reflow things. The fix is in Lomiri, not in the indicators.

ah, got it. we need to fix front end stuff then :) maybe a trigger on rotate
